miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -263,7 +263,13 @@ HotTarget.sample = (function() {
               flat.externalName = sam.externalName;
             },
             pack : function(sam, flat, errorHandler) {
-              sam.externalName = flat.externalName;
+              if (flat.potentialIdentities) {
+                var selectedIdentity = Utils.array.findFirstOrNull(function(item) {
+                  return item.label == flat.identityAlias}, flat.potentialIdentities);
+                if (!selectedIdentity) {
+                  sam.externalName = flat.externalName;
+                }
+              }
```

Comment:
Will that allow new identities to be created again? They're erroring due to `sam.externalName` not getting set.